src/medUtilities/medUtilities.cpp

```diff
@@ -285,6 +285,47 @@ vtkDataArray* medUtilities::getArray(dtkSmartPointer<medAbstractData> data,
     return result;
 }
 
+QList<long> medUtilities::getArrayIndex(dtkSmartPointer<medAbstractData> data,
+                                        QString arrayName)
+{
+    QList<long> indices;
+    int dataType, arrayId;
+    if (data->identifier().contains("vtkDataMesh") ||
+        data->identifier().contains("EPMap"))
+    {
+        vtkMetaDataSet* metaData = static_cast<vtkMetaDataSet*>(data->data());
+        vtkDataSet* mesh = metaData->GetDataSet();
+        // try point data first
+        (void)mesh->GetPointData()->GetAbstractArray(qPrintable(arrayName), arrayId);
```

Comment:
Why the void cast ?

Reply:
To acknowledge that the function has a return value but we chose to ignore it.

Reply:
Actually what I meant was if you are getting a warning.
I'm not really a fan of this personally. There are macros to make this more readable like Q_UNUSED, DTK_UNUSED etc. It's a style issue though so do as you want.

Reply:
(I would personally not put anything :)

Reply:
(i would prefer not having warnings. I don't mind the code style to use)